I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/screens/example_screen_script.rpy", line 7, in script
    call screen example_map_screen
ScriptError: could not find label 'mars_ending'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "/home/gkiss/Misc/renpy-6.99.14.3-sdk/renpy/bootstrap.py", line 306, in bootstrap
    renpy.main.main()
  File "/home/gkiss/Misc/renpy-6.99.14.3-sdk/renpy/main.py", line 513, in main
    run(restart)
  File "/home/gkiss/Misc/renpy-6.99.14.3-sdk/renpy/main.py", line 139, in run
    renpy.execution.run_context(True)
  File "/home/gkiss/Misc/renpy-6.99.14.3-sdk/renpy/execution.py", line 846, in run_context
    context.run()
  File "game/screens/example_screen_script.rpy", line 7, in script
    call screen example_map_screen
  File "/home/gkiss/Misc/renpy-6.99.14.3-sdk/renpy/script.py", line 873, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'mars_ending'.

Linux-4.13.0-38-generic-x86_64-with-debian-stretch-sid
Ren'Py 6.99.14.3.3347
abstract 1.0
Tue Apr 17 09:03:30 2018
